src/containers/BooksForm.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { addBook } from '../actions/index';
+
+
+const CATEGORIES = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      title: '',
+      category: 'default',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    const newState = event.target.id === 'title' ? { title: event.target.value } : { category: event.target.value };
+
+    this.setState(newState);
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+
+    const { addBook } = this.props;
+
+    addBook(this.state);
+
+    this.setState({
+      title: '',
+      category: 'default',
+    });
+  }
+
+  render() {
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <label htmlFor="title">
+          Book Title
+          <input onChange={this.handleChange} id="title" name="title" />
+        </label>
+
+        <label htmlFor="category">
+          Book Category
+          <select onChange={this.handleChange} id="category">
```

Comment:
Maybe you want to update the `value` of the `select` according to the state.

https://reactjs.org/docs/forms.html#handling-multiple-inputs